docs/userGuide/syntax/boxes.mbdf

```diff
@@ -264,6 +296,9 @@ header <hr style="margin-top:0.2rem; margin-bottom:0" /> <small>heading <br> (de
 type | `String` | `'none'` | Supports: `info`, `warning`, `success`, `important`, `wrong`, `tip`, `definition`, or empty for default.
 light | `Boolean` | `false` | Uses a light color scheme for the box.
 seamless | `Boolean` | `false` | Uses a seamless style for the box. If `light` is specified, this style will not be activated.
+no-border | `Boolean` | `false` | Removes border, except if styled by `border-color` or `border-left-color`. 
+no-backgound | `Boolean` | `false` | Removes background, except if styled by `backgound-color` option.
+no-icon | `Boolean` | `false` | Removes icon, except if icon is displayed via `icon` option.
```

Comment:
The attributes list should ideally be sorted. Noted that we have some of the entries being off unfortunately. Perhaps we can move these attributes to above "type" to make it less messy?

Reply:
I see. I assumed we were separating `String` options from the `Boolean` options. I'll move it above "type" for now.